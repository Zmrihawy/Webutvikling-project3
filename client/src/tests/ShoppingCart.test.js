import React from "react";
import shoppingCartPage from "../pages/shoppingCart/ShoppingCartPage";
import renderer from "react-test-renderer";

it("renders shopping cart page", () => {
  const loggedInUser= null;
  const addComponentToShoppingCart = null;
  const removeComponentFromShoppingCart = null;
  const emptyShoppingCart = null;
  const tree = renderer.
  create(<shoppingCartPage
    loggedInUser = {loggedInUser}
    addComponentToShoppingCart = {addComponentToShoppingCart}
    removeComponentFromShoppingCart = {removeComponentFromShoppingCart} 
    emptyShoppingCart = {emptyShoppingCart}/>).toJSON();
  expect(tree).toMatchSnapshot();
});