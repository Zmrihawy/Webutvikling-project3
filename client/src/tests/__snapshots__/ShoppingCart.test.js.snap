// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders shopping cart page 1`] = `
<shoppingCartPage
  addComponentToShoppingCart={null}
  emptyShoppingCart={null}
  loggedInUser={null}
  removeComponentFromShoppingCart={null}
/>
`;
